__mocks__/handlers/self.handler.js

```diff
@@ -2,31 +2,62 @@ import { rest } from 'msw';
 const URL = process.env.NEXT_PUBLIC_BASE_URL;
 
 const selfHandler = [
-	rest.get(`${URL}/users/self`, (_, res, ctx) => {
-		return res(
-			ctx.status(200),
-			ctx.json({
-				id: 'aTOG168A86JXY5wVosJx',
-				github_display_name: 'Mahima Khandelwal',
-				github_id: 'Maheima',
-				roles: {
-					member: true,
-					archived: false
-				},
-				last_name: 'Khandelwal',
-				username: 'mahima',
-				incompleteUserDetails: false,
-				profileStatus: 'BLOCKED',
-				picture: {
-					publicId: 'profile/aTOG168A86JXY5wVosJx/fj2c46kpmpy3gi8tl63s',
-					url: 'https://res.cloudinary.com/realdevsquad/image/upload/v1674639637/profile/aTOG168A86JXY5wVosJx/fj2c46kpmpy3gi8tl63s.jpg'
-				},
-				status: 'active',
-				first_name: 'Mahima',
-				profileURL: 'https://mahima-profile-service.onrender.com'
-			})
-		);
-	}),
+    rest.get(`${URL}/users/self`, (_, res, ctx) => {
+        return res(
+            ctx.status(200),
+            ctx.json({
+                id: 'aTOG168A86JXY5wVosJx',
+                github_display_name: 'Mahima Khandelwal',
+                github_id: 'Maheima',
+                roles: {
+                    member: true,
+                    archived: false,
+                },
+                last_name: 'Khandelwal',
+                username: 'mahima',
+                incompleteUserDetails: false,
+                profileStatus: 'BLOCKED',
+                picture: {
+                    publicId:
+                        'profile/aTOG168A86JXY5wVosJx/fj2c46kpmpy3gi8tl63s',
+                    url: 'https://res.cloudinary.com/realdevsquad/image/upload/v1674639637/profile/aTOG168A86JXY5wVosJx/fj2c46kpmpy3gi8tl63s.jpg',
+                },
+                status: 'active',
+                first_name: 'Mahima',
+                profileURL: 'https://mahima-profile-service.onrender.com',
+            })
+        );
+    }),
```

Comment:
You have changed nothing here so please revert the unnecessary space

Reply:
+1

Reply:
I tried but not happening